LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Jan Schill
```

Comment:
What about us ? :(

Reply:
Was auto-generated, could you research if you can just name all authors comma-separated?